src/store/modules/models.ts

```diff
@@ -13,6 +13,10 @@ export interface ModelItem {
   model_serialized?: object;
 }
 
+export const organism2ModelMapping = {
```

Comment:
Is this the yeast work-around?

Maybe you should open an issue for this too as it seems somewhat hacky and we might forget about it?

Reply:
#41